credential-trust-establishment/spec.md

```diff
@@ -18,118 +18,257 @@ Participate:
 ------------------------------------
 
 ## Introduction
-
-Once trust lists are established, it is natural to extend trust establishment by defining roles and linking them to credentials for situations where
+Once you can create a trust list using Trust Establishment, it is useful to extend Trust Establishment by defining roles and linking them to credentials for situations where:
 1. Enumerating every trusted ecosystem participant in a governance file is not practical (or, in some cases, not possible).
-2. It may be desirable to verify a participant's trusted status via both the trust list entry AND a credential.
+2. It may be desirable to verify a participant's trusted status via a trust list entry AND/OR a credential.
 
-Some of the most important roles are those tied to the issuing of credentials, so it is also necessary to tie roles to credential schemas.
+Some of the most important roles are those tied to the issuing or verifying of credentials, so it is also necessary to tie roles to credential schemas.
 
 Collecting all of this information in one place for ecosystem participants to access is powerful and convenient. A governance file is one relatively simple method for delivering trust establishment information.
 
-Below are three examples of governance files which include sections for trust lists, schemas, roles, and general meta data.
+The following sections cover what is required to enable Credential Trust Establishment:
+- Schemas
+- Roles
+- Linking Participants to Roles
+- Governance File Metadata
+
+## Schemas
+It is very useful to provide a list of schemas which are used in an ecosystem. This list can be quite simple, basically listing schema IDs and a human-friendly name. Here is an example for a U.S.-based higher education ecosystem:
+
+```json
+...
+	"schemas": [
+		{
+			"id": "PdiVKGAjdiVKGAjLqtTroc:2:Accrediting_Body:1.0",
+			"name": "Accrediting Body",
+		},
+		{
+			"id": "BXtzYPyPdiVKGAjLqtPexs:2:University_Degree_Issuer:1.0",
+			"name": "University Degree Issuer",
+		},
+		{
+			"id": "QHqtjywxfZ3yYsFrRHFLQm:2:Bachelors_Degree:1.0",
+			"name": "Bachelors Degree",
+		},
+		{
+			"id": "JAjLqtPexs3yYsFrRHFLQm:2:Student_ID:1.0",
+			"name": "Student ID",
+		}
+	],
+...
+```
+
+## Roles
+Roles are important in an ecosystem because they explain what the trusted ecosystem participants are trusted FOR.
```

Comment:
This is fine. Might want to mention how this helps limit the scope of an interaction to a desired behavior.